dashboard/.eslintrc.yml

```diff
@@ -33,17 +33,17 @@ rules:
   # We use underscores to prefix "private" class methods
   no-underscore-dangle: 0
 
+  import/prefer-default-export: 0
+
   # https://github.com/apollographql/eslint-plugin-graphql
   graphql/template-strings:
     - error
     -
       env: apollo
       schemaJsonFilepath: ./src/schema.json
       validators:
-        - FieldsOnCorrectType
```

Comment:
I realize it's related to [apollo-state-link](https://www.apollographql.com/docs/link/links/state.html#type-checking) but do you mind adding a comment as to why these validators were removed. Or, create an issue to re-add in the future.

This validator in particular is extremely helpful.